examples/gno.land/r/demo/foo20/foo20_test.gno

```diff
@@ -54,3 +54,43 @@ func TestReadOnlyPublicMethods(t *testing.T) {
 		}
 	}
 }
+
+func TestErrConditions(t *testing.T) {
+	admin := std.Address("g1us8428u2a5satrlxzagqqa5m6vmuze025anjlj")
+	empty := std.Address("")
```

Comment:
Maybe add another case?
```go
	invalid := std.Address("invalid")
```

Reply:
grc20 package uses `checkIsValidAddress` to check whether address is valid or not.
https://github.com/gnolang/gno/blob/973be98226366b279ad2ab5931a53d08d0d6138f/examples/gno.land/p/demo/grc/grc20/util.gno#L7-L12

unfortunately it only checks `addr.String() == ""`, so that `invalid` won't be invalid but valid.

Reply:
perhaps better to rename above function to `checkIsEmptyAddress`

Reply:
Ethereum ecosystem libs like Ethers and web3.js commonly utilize the `isAddress` terminology, if the aim is for close parity or to minimize confusion errors for devs

Reply:
Let's handle this in the other refactor #952.

In this particular case, using `isAddress` doesn't seem suitable because we're dealing with an `std.Address` object and checking its validity. 

Creating a more general method like `std.Address{}.IsValid()` would be more appropriate in the future. For now, I'll focus on improving the check in the other PR or add a comment for clarification.

Thank you all for your suggestions.